components/clp-py-utils/clp_py_utils/clp_config.py

```diff
@@ -630,11 +658,14 @@ class ClpConfig(BaseModel):
 
     package: Package = Package()
     database: Database = Database()
```

Comment:
As discussed offline, we should allow users to specify in `clp-config.yaml` which 3rd-party services they want start-clp.sh to start. Otherwise, the behavior is inconsistent because:

* `_set_up_env_for_database` creates directories for database deployment and sets CLP DB config env vars.
* `_set_up_env_for_spider_db` only sets the Spider DB config env vars.

To address this, we're considering adding a `bundled: ["database", "queue", "redis", "results_cache"]` array to `clp-config.yaml` to explicitly define which 3rd-party services should be started. If we do this, we should also create dedicated setup functions for each 3rd-party service deployments so that any deployment-specific initialization logic is cleanly separated.

@coderabiitai create an issue for this